pages/index.js

```diff
@@ -1,20 +1,20 @@
 import useSWR from "swr";
-
+import ArtPieces from "@/components/ArtPieces";
 const fetcher = (url) => fetch(URL).then((response) => response.json());
 
 const URL = `https://example-apis.vercel.app/api/art`;
 
 export default function HomePage() {
   const { data, error, isLoading } = useSWR(URL, fetcher);
-
-  console.log(data);
+  const pieces = data;
```

Comment:
Another way to name the data `pieces` would be to rename the data directly inside the object destructuring like   `const { data: pieces, error, isLoading } = useSWR(URL, fetcher);`